pkg/flow/reconciler/testing/factory.go

```diff
@@ -59,6 +59,7 @@ func MakeFactory(ctor Ctor) rt.Factory {
 		// the controller.Reconciler uses an internal client to handle
 		// component objects
 		ctx, xsltclient := faketminjectionclient.With(ctx, ls.GetXSLTTransformationObjects()...)
+		ctx, jqclient := faketminjectionclient.With(ctx, ls.GetJQTransformationObjects()...)
```

Comment:
This doesn't look right.
We only need one client interface for _all_ flow types, not one _per_ type.

So, instead of "xsltclient" this should be "flowclient".

See how it's done in sources:
https://github.com/triggermesh/triggermesh/blob/0ffd100f029a91d7367a8e5da5192909e3c87402/pkg/sources/reconciler/testing/factory.go#L64